osmosis/README.md

```diff
@@ -24,15 +24,15 @@ import { setupStore } from '@shipt/osmosis';
 
 The `setupStore` function takes in an argument that is just a custom hook. The custom hook will return a single object that represents a slice of state and any functions needed to operate on that state.
 
-`setupStore` returns three variables
+`setupStore` returns one variable with 2 properties: `Context` and `Provider`
 
 ```js
-let [storeContext, wrapperFunction, storeRef] = setupStore(useStateStore);
+let store = setupStore(useStateStore);
 ```
 
-- `storeContext` is a React context variable that gives you access to the state and functions of your store
-- `wrapperFunction` is simply a higher-order component used to provide the store to the app and should be used to wrap the top-level component in the app
-- `storeRef` is an object that gives you access to state variables and functions without causing re-renders when changes occur
+- `store.Context` is a React context variable that gives you access to the state and functions of your store
+- `store.Provider` is simply a higher-order component used to provide the store to the app and should be used to wrap the top-level component in the app
+- `store` is an object that gives you access to state variables and functions without causing re-renders when changes occur
```

Comment:
Lets update the order here to put the `store` first, since that is the main object being returned. Then we can discuss `Context` and `Provider` following `store`. Also, the store description is not accurate (it does cause rerenders). Lets update it to be a bit more generic and clearer...
```
- store - a ref to the store object returned from the supplied custom hook
- store.Context - a React context variable that gives you access to the state and functions of your store
- store.Provider - a higher-order component used to provide the store to the app

```